lib/sms.rb

```diff
@@ -0,0 +1,16 @@
+class Sms
+
+  def initialize
+    @tokens = YAML::load(File.open('./lib/resources/tokens.yml'))
+  end
+
+   def send(delivery_time)
+     @tokens = YAML::load(File.open('./lib/resources/tokens.yml'))
+     client = Twilio::REST::Client.new @tokens["account_sid"], @tokens["auth_token"]
```

Comment:
Metrics/LineLength: Line is too long. [84/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)